CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
 ## 0.1.6
 - removed lombok dependecy
-- improved logging and error messages to caller
\ No newline at end of file
+- improved logging and error messages to caller
+
+## 0.1.7
```

Comment:
I think we should have the newest version in the top for easy access. So flip the order of the changelog.